app/code/Bulbulatory/Recomendations/Ui/Component/Listing/Column/RecommenderEmail.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Bulbulatory\Recomendations\Ui\Component\Listing\Column;
+
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Ui\Component\Listing\Columns\Column;
+use \Magento\Customer\Api\CustomerRepositoryInterface;
+
+class RecommenderEmail extends Column
+{
+    protected $customerRepositoryInterface;
```

Comment:
Tutaj deklarujesz `$customerRepositoryInterface`, a następnie korzystasz z `$_customerRepositoryInterface`. PHP jest jaki jest, nie wywala się przez to i wciąż wszystko działa, więc też łatwo takie rzeczy mogą umknąć, ale fajnie byłoby to poprawić.